hq/app/aws/ec2/EC2.scala

```diff
@@ -2,6 +2,7 @@ package aws.ec2
 
 import aws.AWS
 import aws.AwsAsyncHandler.{awsToScala, handleAWSErrs}
+import aws.support.TrustedAdvisor.refreshTrustedAdvisorCheck
```

Comment:
🤔 this is imported but I don't think it is used?

Reply:
will remove that one.